src/Interactive/CsiCore/Csi.cs

```diff
@@ -17,14 +15,18 @@ internal static int Main(string[] args)
         {
             try
             {
-                var responseFile = Path.Combine(AppContext.BaseDirectory, InteractiveResponseFileName);
+                // Note that AppContext.BaseDirectory isn't necessarily the directory containing csi.exe.
+                // For example, when executed via corerun it's the directory containing corerun.
+                string csiDirectory = Path.GetDirectoryName(typeof(Csi).GetTypeInfo().Assembly.ManifestModule.FullyQualifiedName);
```

Comment:
I'm not sure. The doc on ```AppContext.BaseDirectory``` says 

> Gets the base directory that the assembly resolver uses to probe for assemblies.

That doesn't seem to guarantee that it's the directory ```csi.exe``` is located in.